tests/UnifiedSpecTests/Operation.php

```diff
@@ -254,9 +258,12 @@ private function executeForClient(Client $client)
                 assertArrayHasKey('pipeline', $args);
                 assertIsArray($args['pipeline']);
 
+                $this->sleepUntilNextSecond();
```

Comment:
@alcaeus and I discussed this and he noted that this approach would require additional changes to event monitoring to ignore `startAtOperationTime` in the outgoing `aggregate` command.

An alternative approach would be to conditionally ignore extra `create` events in the change stream, and that's something we can likely do from the spec tests themselves. He's going to pursue that option next.